luaa.h

```diff
@@ -122,6 +122,14 @@ luaA_checkboolean(lua_State *L, int n)
     return lua_toboolean(L, n);
 }
 
+static inline const char *
+luaA_checkstring(lua_State *L, int n)
```

Comment:
I am not sure if this is required, because there wasn't such a function already..

If it makes sense it's probably better to not have it `inline`, because this triggers compiling almost all `.o` files?!